index.html

```diff
@@ -73,8 +73,30 @@
     
     <!-- section SERVICES begin -->
     <section class="container">
-        <div class="row">
-            SERVICES
+        <div  class="row">
+            <div class="services"> Our Services</div>
+            <h1 class="services" > For your very specific industry, we have <u>highly-tailored IT solutions.</u></h1>
+           <div id="serviceContainer">
```

Comment:
O kodel cia ID? Gal mes nepradekim su ID zaist, nes speju poto galim bedu prisidaryt. Gal geriau pakeisk i class

Reply:
sita ID bus tik butent sitam reikalui, ji neturetu itakoti tolimesniam darbui. Be jos layout struktura nesusideda